src/resolvers/createRatingSystemResolver.ts

```diff
@@ -0,0 +1,43 @@
+import mongoose from 'mongoose'
+import { Profile, User } from '../models/user'
+import { systemRating } from "../models/rating";
+import bcrypt from 'bcryptjs'
+import * as jwt from 'jsonwebtoken'
+
+const SECRET = process.env.SECRET || 'test_secret'
+const ratingSystemResolver = {
+    Query: {
+        async getRatingSystem(){
+          const ratingSystem = await systemRating.find({});
+          return ratingSystem;
+        }
+    },
+    Mutation: {
+        async loginUser(_: any, { loginInput: { email, password } }: any) {
+            const user = await User.findOne({ email: email })
+            if (await user?.checkPass(password)) {
+                const token = jwt.sign(
+                    { userId: user._id, role: user._doc?.role || 'user' },
+                    SECRET,
+                    { expiresIn: '2h' }
+                )
+                const data = {
+                    token: token,
+                    user: { ...user._doc, password: '', role: user?.role || 'user' },
+                }
+                return data
+            } else {
+                throw new Error('Invalid credential')
+            }
+        },
```

Comment:
Can you remove this as login and signup have been merged.

Reply:
This needs to be removed